### DISCUSSION TITLE
Fail to build stella vslam installation

### DISCUSSION THREAD

**Discussion by Shrioe** [stella-cv/stella_vslam#608] [Q&A]
Hello, im new to vslam, I'm trying to run the installation guide for stella vslam:
"# Install stella_vslam core library
mkdir -p ~/lib
cd ~/lib
git clone --recursive https://github.com/stella-cv/stella_vslam.git
cd stella_vslam
mkdir build && cd build
cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
make -j4
sudo make install"

Hpwever, the screen will freeze every time I run the 
"make -j4"
Same thing happens when I run make -j -j2 -j3

May I have some suggestions on how to solve this issue, please?

Thank you!

*I'm using raspberrypi 4b, ubuntu 20.04, and realsense 435i

**Comment by The-Real-Thisas** [comment]
-j1
